src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -107,5 +108,35 @@ public void ThrowInInvocationCompletes()
 
             SyntaxFactory.ParseExpression(code);
         }
+
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/13719")]
```

Comment:
I don't believe this is the convention that we were using, WorkItem attribute is always required.